### DISCUSSION TITLE
Enable roundbox

### DISCUSSION THREAD

**Discussion by psi4j** [gokcehan/lf#1724] [Q&A] [answered]
I normally use lf without any boxes drawn around the splits, but I thought I'd give it a try with the rounded corners which would emulate the same effect of Hyprland and could work well with my compositor's aesthetic. I saw that there is a boolean for this setting and I am able to get drawbox working, but not roundbox. Having both set doesn't seem to change the corners from the standard square corners. Is there another setting I need to toggle on in order to get this to work?

Thank you.

**Comment by joelim-work** [comment, marked as answer]
The following config should be sufficient for rounded corners:

```
set drawbox true
set roundbox true
```

I believe you already have this, but `roundbox` is only available on the master branch as it was added just after version [r32](https://github.com/gokcehan/lf/releases/tag/r32).

**Reply by psi4j** [reply]
It works like a charm :]
Thank you.
